hack/verify-bazel.sh

```diff
@@ -18,6 +18,8 @@ set -o nounset
 set -o pipefail
 set -o verbose
 
+if ! which bazel &>/dev/null; then echo "Bazel not available, skipping validation"; exit; fi
```

Comment:
FYI it is possible to install gazelle without bazel, but in general test-infra at least seems to be signalling that bazel may soon be required for developers (which I agree with):
https://github.com/kubernetes/test-infra/blob/master/hack/verify-bazel.sh#L43